### PR TITLE
Add Datto

### LINKED ISSUE #3697: Please add Datto.com
Please consider adding Datto (datto.com) to the list of hosts providing 2FA.

The setup instructions are here:

* https://kb.datto.com/hc/en-us/articles/360021169311-Getting-Started-With-Two-Factor-Authentication-2FA-in-the-Datto-Partner-Portal
* https://kb.datto.com/hc/en-us/articles/360016176791-How-Do-I-Enable-Two-Factor-Authentication-For-My-End-Users-

Supported 2FA methods are:
* SW Token (Authy)
* SMS

### PR DESCRIPTION
**Pull request by Carlgo11**
Resolves #3697 and  #4973.

### CODE REVIEW

File: entries/d/datto.com.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "Datto": {
+    "domain": "datto.com",
+    "tfa": [
+      "totp",
+      "custom-software"
+    ],
+    "custom-software": [
+      "duo"
```
Comment (ApCoder123) [inline entries/d/datto.com.json]:
```suggestion
      "Duo"
```

File: entries/d/datto.com.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    ],
+    "documentation": "https://help.datto.com/s/article/KB360032979352",
+    "keywords": [
+      "remote",
+      "marketing",
```
Comment (ApCoder123) [inline entries/d/datto.com.json]: I don't really see how this fits into marketing?

Comment (Carlgo11) [reply entries/d/datto.com.json]: I saw a marketing service called [MarketNow](https://www.datto.com/partners/marketnow) on their site but it looks like it's some sort of partnering company and not a main service from Datto.

**Review by ApCoder123 — APPROVED** [review]

File: entries/d/datto.com.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      "totp",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Duo"
+    ],
+    "documentation": "https://help.datto.com/s/article/KB360032979352",
+    "keywords": [
+      "remote",
+      "iot"
```
Comment (phallobst) [inline entries/d/datto.com.json]:
```suggestion
      "remote",
      "iot",
      "backup"
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**